bootstrap/bootstrap-storage.sh

```diff
@@ -41,60 +40,57 @@ else
   JAVA="${JAVA_HOME}/bin/java"
 fi
 
-TABLE_INITIALIZER_MAIN_CLASS=com.hortonworks.streamline.storage.tool.TablesInitializer
-for file in "${BOOTSTRAP_DIR}"/lib/*.jar;
+if grep -E "^(\s)*db.type:(\s)*\"mysql\"(\s)*$" $CONFIG_FILE_PATH > /dev/null; then
+  if [ ! -f ../libs/$MYSQL_JAR_PATTERN ] ; then
+    echo "MySQL java connector jar not found in libs, downloading the jar from $MYSQL_JAR_URL_PATH .."
+    TEMP_DIR=`mktemp -d`
+    MYSQL_CONNECTOR_DIR="mysql-connector-java"
+    ZIP_FILE_NAME="$TEMP_DIR/$MYSQL_CONNECTOR_DIR.zip"
+    curl -o $ZIP_FILE_NAME -LOk $MYSQL_JAR_URL_PATH > /dev/null 2>&1
+    unzip $ZIP_FILE_NAME -d $TEMP_DIR/$MYSQL_CONNECTOR_DIR > /dev/null
+    cp $TEMP_DIR/$MYSQL_CONNECTOR_DIR/*/$MYSQL_JAR_PATTERN ../libs
+    rm -rf $TEMP_DIR
+    echo "Successfully copied MySQL connector jar to libs"
+  fi
+fi
+
+for file in "${BOOTSTRAP_DIR}"/../libs/*.jar;
 do
     CLASSPATH="$CLASSPATH":"$file"
 done
 
-echo "Using Configuration file: ${CONFIG_FILE_PATH}"
-
-function dropTables {
-    ${JAVA} -Dbootstrap.dir=$BOOTSTRAP_DIR  -cp ${CLASSPATH} ${TABLE_INITIALIZER_MAIN_CLASS} -m ${MYSQL_JAR_URL_PATH} -c ${CONFIG_FILE_PATH} -s ${SCRIPT_ROOT_DIR} --drop
-}
-
-function createTables {
-    ${JAVA} -Dbootstrap.dir=$BOOTSTRAP_DIR  -cp ${CLASSPATH} ${TABLE_INITIALIZER_MAIN_CLASS} -m ${MYSQL_JAR_URL_PATH} -c ${CONFIG_FILE_PATH} -s ${SCRIPT_ROOT_DIR} --create
-}
-
-function checkStorageConnection {
-    ${JAVA} -Dbootstrap.dir=$BOOTSTRAP_DIR  -cp ${CLASSPATH} ${TABLE_INITIALIZER_MAIN_CLASS} -m ${MYSQL_JAR_URL_PATH} -c ${CONFIG_FILE_PATH} -s ${SCRIPT_ROOT_DIR} --check-connection
-}
-
 function printUsage {
-    echo "USAGE: $0 [create|drop|check-connection|drop-create]"
+    cat <<-EOF
+USAGE: $0 [baseline|migrate|info|validate|clean]
+   baseline : Use this option, if the target database doesn't have DATABASE_CHANGE_LOG table and it already has some migrations.
```

Comment:
Is this the option to be used during a new setup? If so name it "install" and start from "v0". I don't think we need to give option to install from some intermediate baseline option.